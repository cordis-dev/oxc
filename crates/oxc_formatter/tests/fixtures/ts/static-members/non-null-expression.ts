(<IJSONSchema>(
  compoundConfigurationsSchema.items
)).oneOf![1].properties!.folder.enum = folderNames;
