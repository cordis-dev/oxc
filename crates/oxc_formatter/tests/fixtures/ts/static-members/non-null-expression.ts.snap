---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
(<IJSONSchema>(
  compoundConfigurationsSchema.items
)).oneOf![1].properties!.folder.enum = folderNames;

==================== Output ====================
(<IJSONSchema>(
  compoundConfigurationsSchema.items
)).oneOf![1].properties!.folder.enum = folderNames;

===================== End =====================
